vimrc

```diff
@@ -46,12 +50,14 @@ augroup vimrcEx
   autocmd BufRead,BufNewFile .{jscs,jshint,eslint}rc set filetype=json
 
   " ALE linting events
-  set updatetime=1000
-  let g:ale_lint_on_text_changed = 0
-  autocmd CursorHold * call ale#Lint()
-  autocmd CursorHoldI * call ale#Lint()
-  autocmd InsertEnter * call ale#Lint()
-  autocmd InsertLeave * call ale#Lint()
+  if v:version >= 800
```

Comment:
Ah, that emit conflict warnings thing looks like it violates the spirit of this code: https://github.com/w0rp/ale/blob/16772298ab555119223c50a7a8bf4800c436e2f9/plugin/ale.vim#L24-L33